src/jvmMain/kotlin/CommonDefsImplJvm.kt

```diff
@@ -1,15 +1,13 @@
 package org.intellij.markdown.html
 
-import it.unimi.dsi.fastutil.ints.IntArrayList
-import it.unimi.dsi.fastutil.ints.IntStack
+import org.intellij.markdown.lexer.Stack
 import java.net.URLEncoder
-import java.util.Stack
 
 actual class BitSet actual constructor(size: Int): java.util.BitSet(size){
     actual val size = size()
 }
 
-actual typealias IntStack = IntArrayList
+actual class IntStack: Stack<Int>()
```

Comment:
I think if you replace `IntArrayList` with `Stack<Int>`, `actual/expect` classes become redundant and they can be replaced just with `Stack<Int>`. It's declared in a common source set and it's currently used for `js` and `native`.

Reply:
Yes, i chose to leave it since `IntStack` is used in generated code, I wanted to keep the PR smaller. But I can make that change if one of the maintainers wants me to.

Reply:
Makes sense.